api/src/main/java/com/strategyobject/substrateclient/api/RpcSectionModule.java

```diff
@@ -0,0 +1,29 @@
+package com.strategyobject.substrateclient.api;
+
+import com.google.inject.AbstractModule;
+import com.strategyobject.substrateclient.rpc.GeneratedRpcSectionFactory;
+import com.strategyobject.substrateclient.rpc.RpcSectionFactory;
+import com.strategyobject.substrateclient.rpc.registries.RpcDecoderRegistry;
+import com.strategyobject.substrateclient.rpc.registries.RpcEncoderRegistry;
+import com.strategyobject.substrateclient.scale.registries.ScaleReaderRegistry;
+import com.strategyobject.substrateclient.scale.registries.ScaleWriterRegistry;
+import com.strategyobject.substrateclient.transport.ProviderInterface;
+
+public class RpcSectionModule extends AbstractModule {
```

Comment:
How about keeping modules in accordance with packages, and if it's required to have multiple modules per package combined in the proper single one? I think it would be more intuitive seeing all dependencies of the module in one place.